clients/ts/package.json

```diff
@@ -19,6 +19,10 @@
         "enableTsDiagnostics": true
       }
     },
+    "reporters": [
+      "default",
+      [ "./common/node_modules/jest-junit/index.js", { "output": "../../artifacts/logs/signalr.junit.xml" } ]
```

Comment:
Use a `.js` config file (https://jestjs.io/docs/en/configuration.html) and insert the OS into the file name (`../../artifacts/logs/signalr.unit-tests.${process.platform}.junit.xml`)

Ditto for karma, and maybe see what the XML looks like so we can try to get all the browsers to show up properly?